backend/src/project/service.ts

```diff
@@ -31,3 +32,11 @@ export const validateProject = (req: unknown): validProjectOrError => {
         return [false, error];
     }
 };
+
+export const isUserAuthorised = (project: Project, user: string): boolean => {
```

Comment:
Zwracaj uwagę na nazewnictwo zmiennych czy dobrze oddają to co przechowują. Ja np widząc nazwę `user` oczekiwałbym, że to jest cały obiekt usera, natomiast to jest tylko jego username, więc dużo bardziej pasuje tu nazwa `userName`. (To samo można by było powiedzieć o `project.author`, ale to tymczasowe rozwiązanie, docelowo, będziemy tam trzymać id authora i pewnie będzie się joinować z encją usera).